packages/mux-audio-react/src/index.tsx

```diff
@@ -11,9 +11,12 @@ import {
 } from "@mux-elements/playback-core";
 import { getPlayerVersion } from "./env";
 
-export type Props = React.DetailedHTMLProps<
-  React.AudioHTMLAttributes<HTMLAudioElement>,
-  HTMLAudioElement
+export type Props = Omit<
+  React.DetailedHTMLProps<
+    React.AudioHTMLAttributes<HTMLAudioElement>,
+    HTMLAudioElement
+  >,
+  "autoPlay"
```

Comment:
here, we want to use our definition of `autoPlay` from `MuxMediaProps` rather than the native one in React.